src/components/frame/RouteLayouts/UserNavigation/UserNavigation.tsx

```diff
@@ -42,13 +43,22 @@ const UserNavigation = () => {
 
   // const userLock = userData?.user.userLock;
   // const nativeToken = userLock?.nativeToken;
-  // const canInteractWithNetwork = useCanInteractWithNetwork();
+  const canInteractWithNetwork = useCanInteractWithNetwork();
 
   const { userReputation, totalReputation } = useUserReputation(
     colony?.colonyAddress,
     wallet?.address,
   );
 
+  const mockedTokenBalanceData = {
+    nativeToken: colony?.nativeToken,
+    inactiveBalance: BigNumber.from(11 ** 15),
+    lockedBalance: BigNumber.from(0),
+    activeBalance: BigNumber.from(11 ** 15),
+    totalBalance: BigNumber.from(11 ** 15),
+    isPendingBalanceZero: true,
+  };
```

Comment:
Where is this data coming from? Events? Directly from chain?

Reply:
@rdig for now I just mocked the data as I saw in it in the dapp, @jakubcolony is working on the sagas
#286 which should update the data format